Makefile

```diff
@@ -131,7 +131,7 @@ define docker_pull
 endef
 
 define gen-static-stub
-	[ -e ./server/static/files.go ] || echo -e "//DELETEME\npackage static\nimport \"net/http\"\nfunc ServeHTTP(w http.ResponseWriter, r *http.Request) {}\nfunc Hash(file string) string { return \"\" }" > ./server/static/files.go
+	[ -e ./server/static/files.go ] || cp ./server/static/files.go.stub ./server/static/files.go
```

Comment:
you can use `ifneq` in `Makefile` to change targets completely, I think you could avoid function like this:

```
server/static/files.go:
ifeq ($(STATIC_FILES),true)
  // do yarn stuff
else
  cp server/static/files.go.stub server/static/files.go
endif
```

Reply:
I'm thinking that I wouldn't want to use a build flag for this, and I think the current approach gives us more granularity in exactly where and when we create the mock file. Happy to use this pattern if you feel strongly about it though

Reply:
Please, unless you really cannot make it work.

Reply:
Will do

Reply:
Done